src/main.rs

```diff
@@ -1,10 +1,39 @@
+extern crate egg_mode;
+extern crate dotenv;
 extern crate fs2;
 extern crate rand;
+extern crate tokio_core;
 
 mod name_gen;
 mod epic;
 
+use dotenv::dotenv;
+use std::env;
+use name_gen::*;
```

Comment:
Don't ever `use *;`. Explicitly use the items you care about, or namespace the functions when they're used.